azure-pipelines.yml

```diff
@@ -7,11 +7,11 @@ schedules:
     displayName: Nightly build
     branches:
       include:
-        - master
+        - branch-5.15
```

Comment:
I'm not sure if this is really needed on this branch. I don't know how it works.